gitgud/operations.py

```diff
@@ -215,3 +221,20 @@ def get_operator():
         if os.path.isdir(gg_path):
             return Operator(path)
     return None
+
+
+def clear_tracked_commits(file_operator):
+    with open(file_operator.commits_json_path, 'w') as fp:
+        json.dump({}, fp)
+
+
+def track_commit(file_operator, commit_num, commit_hash):
+    # Assumes that .git/gud/commits.json has been initialized by 'git gud load'
+    if os.path.exists(os.path.join(file_operator.git_path, "gud", "commits.json")):
+        with open(os.path.join(file_operator.git_path, "gud", "commits.json")) as fp:
+            commit_dict = json.load(fp)
+        commit_dict["C" + commit_num] = commit_hash
+        with open(os.path.join(file_operator.git_path, "gud", "commits.json"), 'w') as fp:
+            json.dump(commit_dict, fp)
+    else:
+        print("ERROR: Commit tracker does not exist!")
```

Comment:
These seem like the belong in other PRs, is that correct?

Reply:
yes specifically track_commits.